mtp_api/templates/mtp_auth/new_user.html

```diff
@@ -1,29 +0,0 @@
-{% extends 'mtp_common/email_base.html' %}
-{% load i18n %}
-
-{% block content %}
-  <p>
-    {% blocktrans trimmed %}
-      Your new {{ service_name }} account has been created for you.
-    {% endblocktrans %}
-  </p>
-
-  <table cellspacing="10">
-    <tr>
-      <th valign="top" align="left">{% trans 'Username' %}</th>
-      <td>{{ username }}</td>
-    </tr>
-    <tr>
-      <th valign="top" align="left">{% trans 'Password' %}</th>
-      <td>{{ password }}</td>
-    </tr>
-  </table>
-
-  {% if login_url %}
```

Comment:
Not quite sure how this `login_url` could be empty. Maybe some roles have no login URL?
Or maybe it was just a guard against some edge case?

Reply:
It used to be possible for it to be empty, but it's no longer the case (as of a4323f7d2653c4d4a426bdcbe4381eefdd69d84d) but it looks like we didn't remove the unnecessary is-it-blank checks